config/webpack/webpack.config.js

```diff
@@ -218,15 +222,25 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
     resolveModules.unshift(path.resolve(rootPath, 'aggregated-translations'));
   }
 
-  const themeOverride = process.env.THEME; // Flexes root theme for theme visual regression testing.
-  const themeFile = ThemeAggregator.aggregate(themeOverride);
+  const defaultTheme = process.env.THEME; // Flexes root theme for theme visual regression testing.
+  const themeConfig = defaultTheme ? { theme: defaultTheme } : null;
+  let themeFile;
+  if (!disableAggregateThemes) {
+    // Set the default theme and disable scoped theme aggregation.
+    themeFile = ThemeAggregator.aggregate(
+      null,
+      themeConfig,
+      true,
```

Comment:
can we add a comment signifying what this flag is doing?

Reply:
I broke my own rules here, I prefer taking a single object as params so that they can be explicitly name. I'll switch the second param to an object so the API is self documenting.